tests/selenium/pbtest.py

```diff
@@ -259,7 +259,10 @@ def run(self, result=None):
                 if i == nretries - 1:
                     raise
                 else:
-                    print('\nRetrying {} ...'.format(result))
+                    wait_secs = 2 ** i
+                    print('\nRetrying {} after {} seconds ...'.format(
+                        result, wait_secs))
+                    time.sleep(wait_secs)
```

Comment:
It will do that. But I'm not sure that this delay will help anything. Since the delay is between when the browser turns off and on. So it should not effect the test run, assuming the run is well isolated. I just wanted to make that clear, since it may not be what is intended.

If your are fine with that, then I'm happy to merge this.